README.md

```diff
@@ -46,6 +45,40 @@ It correctly bundles React in production mode and optimizes the build for the be
 The build is minified and the filenames include the hashes.\
 Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Deploying your Application
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure-Samples%2Fcommunication-services-web-calling-widget-hero%2Fmain%2Fdeploy%2Fazuredeploy.json)
+
+This repo supports our one click deployment tools. Interact with the button above to deploy our latest bundled release.
+
+### Actions after deployment
+
+Due to the nature of this application meant to interact with Teams Calling Applications we will need to do a few things following the completion of the deployment regarding your Azure Communication Services resource the app will use.
+
+<img src='./media/config-tutorial-screen.png' width='1000'>
+
+You have two different actions you can take when it comes to your `ResourceConnectionString`:
+- Federate the new Azure communication services resource with your teams tenant that is made as part of your deployment. Please see our [documentation](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-teams-call-queue) on how to do that.
+- Replace the `ResourceConnectionString` value for the application in your [Azure Portal](https://ms.portal.azure.com/) with the connection string that is already federated with your desired Teams tenant.
+
+Next you will need to replace in the Application settings the two Teams Calling Application placeholder values created by the deployment:
+- `AutoAttendantId`
+- `CallQueueId`
+
+Once these two values are replaced your application is ready to use!
```

Comment:
nit: comma here so that 'Once these two values are replaced, your application is ready to use!'